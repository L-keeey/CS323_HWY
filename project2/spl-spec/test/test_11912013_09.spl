int add(int n)
{
  return (n + 1 == n);
}
int test_11912013_4()
{
  int k = 4;
  int r = add(k) + (n + 1 == n);
  return r;
}