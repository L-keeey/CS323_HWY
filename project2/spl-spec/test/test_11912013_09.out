Error type 8 at Line 3: function return type and the declared type mismatch.
Error type 7 at Line 8: unmantching operands,such as adding an integer to a structure variable
